app/models/chore.rb

```diff
@@ -1,7 +1,24 @@
 class Chore < ApplicationRecord
+  enum interval_type: { calendar_days: 0, business_days: 1 }
```

Comment:
@jacohen10  - `enum` defines the `.calendar_days?` and `.business_days` methods.